generators/container/templates/container.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import styles from './<%= _.kebabCase(componentName) %>.css';
+
+function mapStateToProps(state) {
+  return {
+
+  };
+}
+
+function mapDispatchToProps(dispatch) {
+  return {
+
+  };
+}
+
+@connect(mapStateToProps, mapDispatchToProps)
```

Comment:
Oh yeah, I really love the `::` bind, it's really sad that it is still on stage 0, but at least we can hope it reaches stage 1, it's not like the proposal was discarded.